### ISSUE TITLE
[0.8.0] Game launching but...

### ISSUE THREAD

**Issue by m0nk210** [Nibre/MotherVR#125]
Hi Nibre, 

first of all thank you for this awesome mod! It's just amazing work!

I recently moved from the 0.6.0 branch to 0.8.0 and encounter a few bugs that i didn't had before.
The biggest is the game boots up as expected after choosing SteamVR under the MotherVR options just fine BUT as soon as i load or start the game it gets stuck during the loading. 
(Black screen with a spinning white circle) And it will keep that way until i open up the steamVR desktop view and click on the alien isolation icon in the task bar / or just click on my desktop itself on the game on the taskbar, which instantly ends the loading screen and let me play.
Same happens when i open up the menu and want to resume - i have to click on the tab on the windwos desktop. Additional i seem to not get the mirror working which might is related to this?

Use Desktop Game Theatre while SteamVR is active is unchecked and Enable the Steam Overlay while in-game is unchecked too.

Additional thought: I cant recenter the view with the vive wands it seems - the steamVR overlay also does not show the option to recenter the view.

kind regards,
m0nk

**Comment by k4pran** [comment]
I'm also getting this issue. When the AI window isn't the active one the game pauses. During the game the AI window will not stay as the active window and so randomly pauses the game.

**Comment by xylo** [comment]
I have the same issue (using SteamVR). However, pressing ALT+Tab after starting the game fixes the hanging for me (you only have to bring AI in foreground again). Thus since there is a workaround it's only a minor issue for me. But so far I did not discover pauses during the game.

**Comment by m0nk210** [comment]
Yes that's the problem. The game does not stay the active window when I use the menu in-game or load save game. I would try to Alt+tab to bring it in the foreground but the problem is I don't have any window of it. 
I however tried 0.6.0 again and I have no issues. The game is a black window with the cursor - I guess like it should. 

Edit:
GTX 1070 , i5 4670 , 16GB DDR3 , WIN 10

**Comment by dday4thedeceiver** [comment]
See if #129 helps
